app/assets/stylesheets/views/_homepage.scss

```diff
@@ -102,16 +102,20 @@
   padding: 0;
 
   @include govuk-media-query($from: desktop) {
-    @include columns($items: 10, $columns: 2, $selector: "li", $flow: column);
+    width: 66.66%;
+    columns: 2;
+    column-gap: 15px;
   }
+}
 
-  & > li {
-    margin: 0 0 govuk-spacing(4) 0;
-    @include govuk-font($size: 19);
-
-    @include govuk-media-query($from: desktop) {
-      margin-bottom: govuk-spacing(2);
-    }
+.homepage-most-active-list__item {
+  margin: 0 0 govuk-spacing(4) 0;
+  @include govuk-font($size: 19);
+  @include govuk-media-query($from: desktop) {
+    break-inside: avoid-column;
+    margin-bottom: 0;
+    padding-top: 0.1875rem;
+    padding-bottom: calc(#{govuk-spacing(2)} - 0.1875rem);
```

Comment:
After much experimentation, I have used `break-inside: avoid-column` and some tweaks to `padding-top` and `padding-bottom` on the list items. This prevents triggering a possible bug in Safari where focused anchors in a multi-column layout cause links to jump between columns.

I wouldn't normally use magic numbers in CSS like this, but the `0.1875rem` value used in the padding is to compensate for the value used for the `text-decoration-thickness` in our global hover states for links (`text-decoration-thickness: max(3px, .1875rem, .12em);`). This value could be tweaked, and may not be "correct" but when added to the top padding, it prevents Safari from rendering one pixel of the focus state in the opposite column.

The bottom margin was removed and the spacing instead added as `padding-bottom` -- this gives some space within the element for the focus/hover states to render without being cropped off, another possible Safari bug.

This isn't a long-term robust solution so we should investigate this further for a possible refactor.